pkg/workload/admissionchecks.go

```diff
@@ -177,9 +174,8 @@ func HasRetryChecks(wl *kueue.Workload) bool {
 
 // HasRejectedChecks returns true if any of the workloads checks is Rejected
 func HasRejectedChecks(wl *kueue.Workload) bool {
-	for i := range wl.Status.AdmissionChecks {
-		state := wl.Status.AdmissionChecks[i].State
-		if state == kueue.CheckStateRejected {
+	for _, ac := range wl.Status.AdmissionChecks {
```

Comment:
nit: I prefer to revert to avoid distractions

Reply:
Done